src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
```

Comment:
it is default property

Reply:
ok, removed.

Reply:
I don`t understand

Reply:
if you delete this style that nothing changes, so you should delete this

Reply:
also `font-family` should be the same for this two, explained here 
![image](https://user-images.githubusercontent.com/53611910/197050195-bb05f621-62e8-4393-9bd4-fd0fd9a98148.png)

Reply:
+ move `font-face` and general `html` styles into separate file